### PR TITLE
Minimal `DataViewerApp` part 1-1: Histogram

### PR DESCRIPTION
**Pull request by kangz12345**
See #144 for details.

### CODE REVIEW

**Comment by kangz12345** [comment]
I disabled the pylint warning "too-many-statements" for `DataPointWidget.__init__()` since I don't find a good way to reduce the number of statements keeping the code readable and consistent.

File: iquip/apps/dataviewer.py [hunk 1]
```diff
@@ -316,13 +318,23 @@ def __init__(self, parent: Optional[QWidget] = None):
...
-            layout.addWidget(QLabel(":", self), dataType, 2)
-            layout.addWidget(spinbox, dataType, 3)
+            layout.addWidget(button, dataType, 2)
+            layout.addWidget(QLabel(":", self), dataType, 3)
+            layout.addWidget(spinbox, dataType, 4)
+        layout.setColumnStretch(1, 1)
         self.setDataType(DataPointWidget.DataType.P1)
+        # histogram viewer
+        self.histogram = HistogramViewer(title="Photon count histogram",
+                                         labels={"left": "#samples"})
```
Comment (BECATRUE) [inline iquip/apps/dataviewer.py:328]: Is it correct that in order to make the bottom label in [this](https://github.com/snu-quiqcl/iquip/pull/151#issuecomment-1724835467), we need to set `labels={..., "bottom": "Photon count"}`?

Comment (kangz12345) [reply iquip/apps/dataviewer.py:328]: It will be set when we call `setData()`!

Comment (BECATRUE) [reply iquip/apps/dataviewer.py:328]: Now, I understand it thanks for Jiyong's explanation!

**Review by BECATRUE — APPROVED** [review]

**Pull request merged**